docker web_1 startup issue

This is the output I get:
```
web_1      | PUT bucket code: 000
web_1      | Done!
web_1      | yarn run v1.5.1
web_1      | $ node scripts/dev-server.js
web_1      | module.js:549
web_1      |     throw err;
web_1      |     ^
web_1      | 
web_1      | Error: Cannot find module 'chalk'
web_1      |     at Function.Module._resolveFilename (module.js:547:15)
web_1      |     at Function.Module._load (module.js:474:25)
web_1      |     at Module.require (module.js:596:17)
web_1      |     at require (internal/module.js:11:18)
web_1      |     at Object.<anonymous> (/code/scripts/dev-server.js:2:15)
web_1      |     at Module._compile (module.js:652:30)
web_1      |     at Object.Module._extensions..js (module.js:663:10)
web_1      |     at Module.load (module.js:565:32)
web_1      |     at tryModuleLoad (module.js:505:12)
web_1      |     at Function.Module._load (module.js:497:3)
web_1      | error An unexpected error occurred: "Command failed.
web_1      | Exit code: 1
web_1      | Command: sh
web_1      | Arguments: -c node scripts/dev-server.js
web_1      | Directory: /code
web_1      | Output:
web_1      | ".
web_1      | info If you think this is a bug, please open a bug report with the information provided in "/code/yarn-error.log".
web_1      | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
voice-web_web_1 exited with code 1
```
I have zero experience with modern JavaScript/TypeScript, and almost no experience with docker, so maybe I am doing something obviously stupid.

I have the latest version of the voice-web repo.
> npm --version
5.6.0
> docker --version
Docker version 18.03.1-ce, build 9ee9f40
> yarn --version
1.6.0

Let me know if you need further information.

Here are the steps I followed to setup voice-web locally:
- clone the repository
- go to the voice-web folder
- docker-compose up

When running:

`> docker-compose build --no-cache
> docker-compose up -d web s3proxy`

I can see in the logs that we still have the same issue:

`web_1      | Checking if s3 bucket exists...
web_1      | GET bucket code: 000
web_1      | Bucket does not exist. Creating one...
web_1      | PUT bucket code: 000
web_1      | Done!
web_1      | yarn run v1.6.0
web_1      | $ node scripts/dev-server.js
web_1      | module.js:549
web_1      |     throw err;
web_1      |     ^
web_1      | 
web_1      | Error: Cannot find module 'chalk'
web_1      |     at Function.Module._resolveFilename (module.js:547:15)
web_1      |     at Function.Module._load (module.js:474:25)
web_1      |     at Module.require (module.js:596:17)
web_1      |     at require (internal/module.js:11:18)
web_1      |     at Object.<anonymous> (/code/scripts/dev-server.js:2:15)
web_1      |     at Module._compile (module.js:652:30)
web_1      |     at Object.Module._extensions..js (module.js:663:10)
web_1      |     at Module.load (module.js:565:32)
web_1      |     at tryModuleLoad (module.js:505:12)
web_1      |     at Function.Module._load (module.js:497:3)
web_1      | error Command failed with exit code 1.
web_1      | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

yay i reproduced it:
```
db_1       | 2018-06-20 10:51:17 1 [Warning] 'proxies_priv' entry '@ root@4455abae7d51' ignored in --skip-name-resolve mode.
web_1      | GET bucket code: 404
web_1      | Bucket does not exist. Creating one...
web_1      | PUT bucket code: 200
web_1      | Done!
web_1      | yarn run v1.6.0
web_1      | $ node scripts/dev-server.js
web_1      | module.js:549
web_1      |     throw err;
web_1      |     ^
web_1      | 
web_1      | Error: Cannot find module 'chalk'
web_1      |     at Function.Module._resolveFilename (module.js:547:15)
web_1      |     at Function.Module._load (module.js:474:25)
web_1      |     at Module.require (module.js:596:17)
web_1      |     at require (internal/module.js:11:18)
web_1      |     at Object.<anonymous> (/code/scripts/dev-server.js:2:15)
web_1      |     at Module._compile (module.js:652:30)
web_1      |     at Object.Module._extensions..js (module.js:663:10)
web_1      |     at Module.load (module.js:565:32)
web_1      |     at tryModuleLoad (module.js:505:12)
web_1      |     at Function.Module._load (module.js:497:3)
web_1      | error Command failed with exit code 1.
web_1      | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
voice-web_web_1 exited with code 1
```

--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -24,4 +24,4 @@ services:
       - s3proxy
     ports:
       - 9000:9000
-    command: bash -c "/code/docker/prepare_s3.sh && yarn start"
+    command: bash -c "/code/docker/prepare_s3.sh && yarn && yarn start"

